internal/pkg/skuba/kubernetes/versions.go

```diff
@@ -115,7 +115,7 @@ var (
 				Tooling:   &ContainerImageTag{Name: "skuba-tooling", Tag: "0.1.0"},
 			},
 			AddonsVersion: AddonsVersion{
-				Cilium:  &AddonVersion{"1.5.3", 2},
+				Cilium:  &AddonVersion{"1.6.6", 1},
```

Comment:
```suggestion
				Cilium:  &AddonVersion{"1.6.6", 3},
```

The manifest version should always be bumped, is not local to the version, but globally.